### PR TITLE
Update the failing import

### PR DESCRIPTION
**Pull request by jwnasambu**
## Requirements

- [ ] This PR has a title that briefly describes the work done including the ticket number. If there is a ticket, make sure your PR title includes a [conventional commit](https://o3-docs.openmrs.org/docs/frontend-modules/contributing.en-US#contributing-guidelines) label. See existing PR titles for inspiration.
- [ ] My work conforms to the [OpenMRS 3.0 Styleguide](https://om.rs/styleguide) and [design documentation](https://om.rs/o3ui).
- [ ] My work includes tests or is validated by existing tests.

## Summary
I resolved the import error on the Patient chat due to `getPatientChartStore`  which is not exported from the @openmrs/esm-patient-common-lib.

## Screenshots
**Before fixing the issue**

Screenshot 2024-11-04 at 5 47 58 PM

**After fixing the import**

Screenshot 2024-11-04 at 6 00 20 PM

### CODE REVIEW

**Comment by jwnasambu** [comment]
@denniskigen, @ibacher kindly help me review my PR at your convenient time please!

File: packages/esm-patient-chart-app/src/patient-chart/patient-chart.component.tsx [hunk 1]
```diff
@@ -17,7 +17,7 @@ import Loader from '../loader/loader.component';
 import styles from './patient-chart.scss';
 import VisitHeader from '../visit-header/visit-header.component';
 import SideMenuPanel from '../side-nav/side-menu.component';
-import { getPatientChartStore } from '@openmrs/esm-patient-common-lib';
+import { getPatientChartStore } from '@openmrs/esm-patient-common-lib/src/store/patient-chart-store';
```
Comment (denniskigen) [inline packages/esm-patient-chart-app/src/patient-chart/patient-chart.component.tsx:20]: We shouldn't need this change, right, @ibacher? `getPatientChartStore` should be resolved from automatically from the public API of `@openmrs/esm-patient-common-lib` without needing to import it deeply like so.

**Comment by denniskigen** [comment]
Closing owing to the explanation above. Thanks!

File: packages/esm-patient-chart-app/src/patient-chart/patient-chart.component.tsx (hunk 1 above)
Comment (ibacher) [reply packages/esm-patient-chart-app/src/patient-chart/patient-chart.component.tsx:20]: Yes. In fact, deep imports like this have the potential to break the whole shared dependencies structure.

Comment (ibacher) [reply packages/esm-patient-chart-app/src/patient-chart/patient-chart.component.tsx:20]: (The only place we do really do things like this is with esm-framework, and there's a lot of configuration around ensuring that works).

**Pull request closed without merging**